packages/editor/src/pages/Editor/index.tsx

```diff
@@ -63,3 +66,55 @@ class Editor extends Component<{}, IState> {
 }
 
 export default Editor;
+
+///////////////////////////////////////
+
+async function ensureProperOfficeBuildIfRelevant() {
```

Comment:
Are these ever going to be used elsewhere? / Should the be factored out to a utilities file somewhere?